src/utils/response.ts

```diff
@@ -0,0 +1,12 @@
+export interface LambdaResponse {
```

Comment:
This isn't needed. This file should be just a utils file for batch item failures, it doesn't cover all responses (nor does it need to)